server/src/domain/ldap/group.rs

```diff
@@ -20,6 +20,8 @@ use crate::domain::{
     types::{AttributeName, AttributeType, Group, GroupId, LdapObjectClass, UserId, Uuid},
 };
 
+pub const DEFAULT_GROUP_OBJECT_CLASSES: &[&str] = &["groupOfUniqueNames"];
```

Comment:
Can we make that not pub, and instead expose a function that returns a `Vec<Vec<u8>>`? That'll simplify the rest of the code.
And maybe another one that returns a vec of LDAP object class